<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThrustCurveMotorSelector</class>
 <widget class="QDialog" name="ThrustCurveMotorSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>878</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>160</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Manufacturer</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Diameter</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Impulse Class</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="manufacturer"/>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="diameter"/>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="impulseClass"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="getMetadata">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Get Metadata</string>
   </property>
  </widget>
  <widget class="QCustomPlot" name="plot" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>601</width>
     <height>551</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>210</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="1">
     <widget class="QComboBox" name="motorSelection"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="setMotor">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>290</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>setMotor</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>gui/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
